app/src/main/java/com/sopt/now/main/MainActivity.kt

```diff
@@ -0,0 +1,62 @@
+package com.sopt.now.main
+
+import android.os.Bundle
+import android.util.Log
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import com.sopt.now.R
+import com.sopt.now.databinding.ActivityMainBinding
+import com.sopt.now.main.homeFragment.HomeFragment
+import com.sopt.now.main.myPageFragment.MyPageFragment
+import com.sopt.now.main.searchFragment.SearchFragment
+
+class MainActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityMainBinding
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        val memberId = intent.getStringExtra("memberId")
```

Comment:
사소한 것이지만 변수를 선언할 때 해당 변수의 타입을 명시 해주는 것이 좋습니다.
```suggestion
        val memberId : String = intent.getStringExtra("memberId")
```

Reply:
사소한 디테일 감사합니다!!